### ISSUE TITLE
Memory leak

### ISSUE THREAD

**Issue by mnaiman** [kee-org/browser-addon#252]
There is terrible memory leak. Some 493453 bytes icon is duplicated in memory 9684+9683 times.
This makes my Firefox eats 24GB RAM and counter is growing until Firefox crash

There is some help where to start searching

![image](https://user-images.githubusercontent.com/2264442/74595945-658f9f00-5048-11ea-912d-775e7efb8a90.png)

Great addon BTW!

**Comment by mnaiman** [comment]
Cannot make replies on forum because im new user, so reply here:
Every restart of browser, it focuses on another 2 icons, so it is not site specific :/

**Comment by mnaiman** [comment]
In settings I have checked not to work with favicons - no effect

**Comment by rjt** [comment]
@mnaiman How did you get to that "**Explicit Allocations**" screen?   
Is it using the builtin Firefox F12 debug functionality?  If you could tell us how to reproduce or point us to a blog post or something, that would help me.  Firefox has been gobbling my 32GB of RAM.

**Comment by mnaiman** [comment]
about:memory to URL
Click on Measure
Filter d9e6bc59-5bd4-45a1-b11f-e0f90ec7b5ef/_generated_background_page.html
Boom :)

It needs some time (1-2days for plugin to eat lot of memory)
I do one snapshot after start of firefox and them measures in time and compare difference

**Comment by Drive4ik** [comment]
@mnaiman 
I confirm.
Windows 10 64bit, FF stable 75.0
I left FF for the night, and by morning all RAM was full (16 GB). All tabs were unloaded from memory. I saw things in add-ons, started turning them off and on without restarting the browser, and found this add-on.
I want to clarify what the addon installed from github
`kee_password_manager-3.3.12beta-an + fx.xpi`
I had to reinstall from FF AMO.
I did not install version 3.4+, maybe this bug was fixed in it.

@luckyrat 
Great addon!
